### PR TITLE
Update cd.md

### PR DESCRIPTION
**Pull request by rogerxaic**
OpenShift Operator-deployed ArgoCD does no longer feature a "Log in with OpenShift" button. Here are instructions on how to retrieve the `admin` password and how to use it to log in to ArgoCD.

### CODE REVIEW

**Comment by lsteck** [comment]
Hi @rogerxaic 

Thanks for the update but I think it would be easier to run the `oc credentials` command to get the ArgoCD admin password.  Could you update your changes to use that command?  Thanks!

File: docs/resources/workshop/cd.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ Promote an Application using CD with GitOps and ArgoCD
     ```
 
 1. Verify Application is deployed in **QA**
-    - Select ArgoCD from the Console Link and login using OpenShift login
+    - Get ArgoCD's `admin` password by running `oc extract secrets/openshift-gitops-cluster --keys=admin.password -n openshift-gitops --to=-`
```
Comment (csantanapr) [inline docs/resources/workshop/cd.md:31]: The password is two different places depending OCP46 or OCP47+

OCP46: `secrets/argocd-cluster-cluster`
OCP47: `secrets/openshift-gitops-cluster`

We are making fixes to have it easier for the user to just run `oc credentials`
WIth OCP48 the OpenShift SSO would be fix and user can just click the "Login with OpenShift"

**Review by csantanapr — CHANGES_REQUESTED** [review]
Thanks for @rogerxaic for the PR.
I was able to fix argocd on OCP47 here is the final fix https://github.com/cloud-native-toolkit/ibm-garage-iteration-zero/pull/355

Please update the text to tell the user to use `oc credentials` to get the password

**Review by lsteck — COMMENTED** [review]
LGTM

**Review by csantanapr — APPROVED** [review]

**Pull request merged**